src/graphql/error-map.ts

```diff
@@ -105,7 +106,14 @@ export const mapError = (error: ApplicationError): CustomApolloError => {
         "Too many onchain addresses creation, please wait for a while and try again."
       return new TooManyRequestError({ message, logger: baseLogger })
 
+    case "UnknownRouteNotFoundError":
     case "UnknownLnInvoiceDecodeError":
```

Comment:
I don't think the message for UnknownLnInvoiceDecodeError should be `""Unable to find a route for payment"`